data/js/server/network/helpers/openpaperdoll.js

```diff
@@ -0,0 +1,14 @@
+//usage: TriggerEvent( 18000, "OpenPaperdoll", socket, targChar )
+
+function OpenPaperdoll( socket, targChar )
+{
+    // Create packet to open a paperdoll
+    var pStream = new Packet;
+    pStream.ReserveSize( 66 );
+    pStream.WriteByte( 0, 0x88 );
+    pStream.WriteLong( 1, targChar.serial );
+    pStream.WriteString( 5, targChar.name + ( targChar.title ? " " + targChar.title : "" ), targChar.name.length + ( targChar.title ? targChar.title.length + 1 : 0 ));
```

Comment:
This needs to respect uox.ini settings for what to show in paperdolls, whether it's for a player or an NPC. Basically, to be a useful helper function, it needs to replicate the functionality provided via the following function in cPlayerAction.cpp:

> void PaperDoll( CSocket *s, CChar *pdoll )

Alternatively, if you need this just for the mannequin, include this in the mannequin script instead of a separate helper script. Custom packets can be sent in regular scripts, they don't need to be "network scripts" for that.